service-base/src/main/java/org/eclipse/hono/service/metric/MicrometerBasedMetrics.java

```diff
@@ -127,7 +138,7 @@ public final void incrementConnections(final String tenantId) {
         Objects.requireNonNull(tenantId);
         gaugeForTenant(METER_CONNECTIONS_AUTHENTICATED, this.authenticatedConnections, tenantId, AtomicLong::new)
                 .incrementAndGet();
-        this.totalCurrentConnections.incrementAndGet();
+        trackDeviceConnectionTime(tenantId, this.totalCurrentConnections.incrementAndGet());
```

Comment:
tare you sure that you want to pass in the overall number of connected devices across all tenants? I think you'd want to pass in the value returned by the previous statement, wouldn't you?

Reply:
My bad, I will correct it.